.github/workflows/CI.yml

```diff
@@ -170,11 +171,16 @@ jobs:
                ${{ matrix.extra }}
                if [[ -n ${NO_COVERAGE} ]] ; then
                   echo "No coverage reporting"
-               else
-                  echo "Coverage reporting, updating compiler and linker flags"
+               elif [[ -n ${SAFE_COVERAGE_FLAGS} ]] ; then
+                  echo "Using safe (slower) coverage options, updating compiler and linker flags"
```

Comment:
I find it quite confusing to call this "safe" as these are the flags which broke our CI together with `-pthread` ;-). How about "SLOW" or "FULL" or so?

Reply:
Yes, full is good. I was trying to get across the default is very linux/gcov specific.